mscore/prefsdialog.ui

```diff
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>712</width>
```

Comment:
Apparently Qt Creator did some change behind your back. Could you reset the size and the default tab?
Also remove the & in the strings?